src/inventory.java

```diff
@@ -0,0 +1,122 @@
+import java.util.Arrays;
+
+public class inventory {
```

Comment:
By convention, we usually start class names with a Capital letter. 
```suggestion
public class Inventory {
```